### ISSUE TITLE
Support slice drillthrough for Graph widgets (barchart, colchart, stacked barchart/colchart, pie chart)

### ISSUE THREAD

**Issue by evshvarov** [intersystems-community/DeepSeeWeb#272]
It drills to all the level, but need to the slice level

**Comment by evshvarov** [comment]
Change the behaviour to drillthrough (if there is a listing), and move drilldown to another place - at headers.
<img width="1201" alt="Screenshot 2022-10-28 at 11 22 37" src="https://user-images.githubusercontent.com/2781759/198674691-6ed4a343-6ad8-4741-b1d5-1d4e49d64a32.png">

**Comment by evshvarov** [comment]
[Working sample](https://pie-chart-bug.demo.community.intersystems.com/dsw/index.html#/IRISAPP/Basic%20Dashboard%20Demo.dashboard)

**Comment by Lena-Ev** [comment]
dashboard request
control - show listing means - custom listing

**Comment by gnibeda** [comment]
there is no presented widget in the working sample

**Comment by evshvarov** [comment]
@gnibeda it is there - just switch view from table to graph back:

**Comment by evshvarov** [comment]
![DeepSeeWeb](https://user-images.githubusercontent.com/2781759/201213236-15d5534e-11f5-4d91-9afa-a67cd53c165d.png)

**Comment by Lena-Ev** [comment]
1 - by the click to the axis value - open drilthrough

**Comment by evshvarov** [comment]
You mean drill down, right?

**Comment by Lena-Ev** [comment]
@evshvarov yes I have edited the comment for drill down description only

**Comment by Lena-Ev** [comment]
This feature will be released this week without bug fixing, but testing needed urgently for that version,
Only after testing we can  make new releases

**Comment by evshvarov** [comment]
Works for the majority of cases. But there is a glitch. Drillthrough doesn't work if already drilled down
https://www.loom.com/share/2a757ba2eeea4e6b97422d26aaad1ae6

**Comment by gnibeda** [comment]
Fixed issue when performing drill a second time.

> Works for the majority of cases. But there is a glitch. Drillthrough doesn't work if already drilled down

mdx2json returns empty data for this MDX:
`DRILLTHROUGH  WITH  MEMBER [MEASURES].[Revenue as Percent of Total] AS 'Measures.[Amount Sold] / %MDX(\"select from holefoods\",\"%CONTEXT\",\"filters|columns\")',FORMAT_STRING='##.##%;;;;' MEMBER [MEASURES].[Revenue Percent Cumulative] AS 'Measures.[Revenue as Percent of Total] + %CELL(0,-1)',FORMAT_STRING='##.##%;;;;' SELECT {[Measures].[Amount Sold],[MEASURES].[REVENUE PERCENT CUMULATIVE]} ON 0,NON EMPTY ORDER([Product].[P1].[Product Name].&[SKU-199].children,Measures.[Amount Sold],BDESC) ON 1 FROM [HOLEFOODS] %FILTER [Product].[P1].[Product Category].&[Snack] %FILTER [Product].[P1].[Product Name].&[SKU-199]`

**Comment by evshvarov** [comment]
works now. closing.

**Comment by evshvarov** [comment]
But drillthrough doesn't work after first drilldown.
Video: https://www.loom.com/share/2a757ba2eeea4e6b97422d26aaad1ae6
and [example](https://pie-chart-bug.demo.community.intersystems.com/dsw/index.html#/IRISAPP/Pareto%20Charts/Pareto%20Chart%20for%20Category.dashboard) @gnibeda

**Comment by Lena-Ev** [comment]
The last comment is taken to the new task #321

**Comment by Lena-Ev** [comment]
Reopen, there will be new example from Eugeny

**Comment by evshvarov** [comment]
[Reproduced the case](https://pie-chart-bug.demo.community.intersystems.com/dsw/index.html#/IRISAPP/Bug/Stacked%20Chart%20Drill%20Through.dashboard)

Sliced Drillthrough doesn't work when filters are applied

**Comment by gnibeda** [comment]
Done in 3.1.63

**Comment by evshvarov** [comment]
works!
